src/webview-contract/webviewDefinitions/retinaCapture.ts

```diff
@@ -0,0 +1,19 @@
+import { WebviewDefinition } from "../webviewTypes";
+
+export interface InitialState {
+    selectedNode: string;
+    clusterName: string;
+    retinaOutput: string[];
+    allNodes: string[];
+    captureFolderName: string;
+}
+
+export type ToVsCodeMsgDef = {
+    retinaCaptureResult: string;
```

Comment:
See comments in `RetinaCapturePanel.ts` - some of these should probably be renamed/removed.